storage/rocksdb/rdb_datadic.h

```diff
@@ -387,7 +387,11 @@ class Rdb_key_def
     SECONDARY_FORMAT_VERSION_INITIAL= 10,
     // This change the SK format to include unpack_info.
     SECONDARY_FORMAT_VERSION_UPDATE1= 11,
-    SECONDARY_FORMAT_VERSION_LATEST= SECONDARY_FORMAT_VERSION_UPDATE1,
```

Comment:
What if there is an affected index somewhere that happens not to be used for reads, and so there are no complaints?
The data in the index is corrupted, and will remain so if not modified. 

Then, somebody might eventually write a query that uses the affected index. They will get an error.  If this patch bumps the index number, it will be easier to catch that case such case (by old index number).

I'm not sure whether the above reasoning has much merit in production